<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<link rel="stylesheet" href="css/reset.css">
	<link rel="stylesheet" href="css/style.css">
	<title>TheScientist</title>
</head>
<body>
	<header id="top">
		<div>
			<h1>Genome Spotlight: Edith's Checkerspot Butterfly (Euphydryas editha)</h1>
			<span class="mark">A high-quality genome sequence for this versatile insect will likely aid eco-evolutionary research.</span>
		</div>
	</header>
	<main>
		<div class="lorem">
			<p>Edith's checkerspot butterflies are found throughout western North America, but they don't all look or act the same. Dozens of habitat-specific forms or ecotypes of the species have been documented, and each is so strongly adapted to its local environment that it can differ markedly from populations a mere 20 kilometers away.</p>
			<p class="line">Indeed, decades of laboratory and field experiments on the species' adaptability—including work that garneredStanford population biologist Paul Ehrlich the coveted Crafoord Prize—indicate that this little lepidopteran can make major ecological shifts, such as switching host plant species, remarkably quickly. This makes them excellent models for studying the consequences of anthropogenic activity, but a lack of genomic resources has made such work difficult. A high-quality genome for the species, recently assembled and published July 25 in <a href="https://academic.oup.com/gbe/article/14/8/evac113/6649524?login=false" target="_blank">Genome Biology and Evolution</a>, will likely change that.</p>
				<img src="img/butterfly.jpg" width="900" height="600" alt="butterfly">
			<div class="block">
				<p>To construct the butterfly's 0.6 Gb genome, the team employed Oxford Nanopore long-read sequencing along with Hi-C chromosome capture and Illumina short-read polishing. 
				That resulted in a highly <span class="inside">contiguous assembly that contains 97.5 percent of expected single-copy genes based</span> on a database of previously published lepidopteran genomes.
				Analyses of the sequence revealed 23,870 genes, 20,771 of which the researchers were able to functionally annotate.</p>
				<p class="multishadow">This intel will finally allow scientists to probe the genetic basis of this butterfly's versatility and adaptability. 
				“The E. editha genome, annotation, and functional descriptions now fill a missing gap for one of the leading field-based ecological model systems in North America,” 
				the study authors write in their paper. It should also assist <span class="inside">conservation efforts for the butterfly's endangered subspecies and the species as a whole,</span> which recent research suggests is at risk due to climate change.</p>
			</div>
		</div>
		<div class="curent">
			<h2>Runners Up:</h2>
			<span class="abc">Bald notothen (Trematomus borchgrevinki)</span>
				<p>The bald notothen isn't a particularly charismatic fish. And as a small Antarctic species usually found lurking under ice, it isn't sought-after by fishers, either. 
					This relative obscurity meant there was a dearth of genetic information available for the species and its close relatives—and therefore, it was a great test subject for comparing de novo genome assembly methods.</p>
				<p>So, for a July 29 paper in G3 Genes|Genomes|Genetics, researchers assembled the fish's genome using three tactics: purely with short reads, with a combination of short and long reads, and purely with long reads. 
					And while they identified key sources of error in all three approaches, “long-read contig assembly is the current best choice,” the authors write, as the quality of the resulting assembly was unmatched by those of the other methods.</p>
				<span >Many-banded krait (Bungarus multicinctus)</span>
				<p class="over">The many-banded krait is one of the most venomous snakes in Asia, and famously caused the death of beloved California Academy of Sciences curator and herpetologist Joe Slowinski. In China, the snake is responsible for nearly 1 in 10 venomous bites and has the highest case-fatality rate, killing up to one-third of the time. 
					Developing better treatments requires in-depth knowledge of the animal's toxin repertoire—which is why researchers from the Chinese Academy of Sciences generated a high-quality reference genome for the species. </p>
				<p> The assembly, published July 12 in Cell Reports, revealed that the snake has an impressive repertoire of 118 toxin genes from 17 families. 
					“Our B. multicinctus chromosome-scale assembly and associated transcriptome data provide a valuable resource for exploring the origin and evolution of venom genes, thereby enabling venom-driven drug discovery,” the authors write.</p>
			</div>
	</main>
	<footer>
		<h3>Article by:</h3>
		<span class="mark">
			Christie Wilcox
			Aug 25, 2022</span>
			<div class="link">
			<p><a href="#top">&#11014;UP&#11014;</a></p>
		</div>
	</footer>
</body>
</html>
